registry/polkadot.json

```
@@ -1,10 +1,10 @@
 [
     {
-        "address": "13ActcpzR19JS9TBvBGi2TH3D8sqycVx4e9TebV9MJKHTQ2R",
-        "name": "ChaosDAO",
-        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/chaosdao.png",
-        "shortDescription": "Community org with a dedicated internal voting process and multisig for OpenGov referenda",
-        "longDescription": "## About us\nChaosDAO is a curated community of Polkadot & Kusama natives. Our members are involved with every aspect of the ecosystem as developers, community organizers, [Twitter](https://twitter.com/ChaosDAO) and YouTube educators, and as ordinary retail investors.\n\n## Voting Process\nAs part of our own organization’s evolution we have developed a robust [internal governance mechanism](https://github.com/ChaosDAO-org/ChaosDAO_docs/blob/main/governance/OpenGov.md) to determine how our community will publicly vote on referenda. By voting on behalf of the long-time Dotsama community we strive to ensure that all votes we cast reflect the true best interests of our native ecosystem.\n\n![ChaosDAO Manifesto image](https://raw.githubusercontent.com/ChaosDAO-org/ChaosDAO_docs/main/.gitbook/assets/ChaosDAO_Manifesto_600px.jpg)",
-        "isOrganization": true
+        "address": "15vvAhHcmop8PjP5QNr3mXzX1oesv6vjun25iSgMe9YpDezv",
+        "name": "RobinHodl",
+        "image": "https://github.com/robinhodl69/opengov-delegate-registry/blob/master/images/RobinHodl.png",
```

Also, url should be in that format:
```suggestion
        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/RobinHodl.png",
```